packages/react-native-editor/ios/GutenbergDemo/GutenbergViewController.swift

```diff
@@ -396,6 +397,15 @@ extension GutenbergViewController {
                 self.toggleHTMLMode(action)
         })
     }
+
+    var removeAllBlocksAction: UIAlertAction {
+        return UIAlertAction(
+            title: "Remove All Blocks",
+            style: .default,
+            handler: { [unowned self] action in
```

Comment:
```suggestion
            handler: { action in
```

I think the `unowned self` is unnecessary because the alert, `let alert = UIAlertController...`, is not retained by `self`. To quote [this](https://stackoverflow.com/a/34931631/1305067) StackOverflow answer _'The key question to ask yourself is if your alert object is "owned" by self'_. That's to say, if alert is not a stored property of `self`, `unowned self` is not required.

In this case, the alert is local to the function it's declared in, so it's not retained by `self`. The alert action is not retained either (it's a computed property) so its handler closure can happily retain (keep `self` around) until the view controller that presented the alert is dismissed. 

I don't think there's a danger of the app crashing if we leave things as-is (this is also just the demo app), but it seems good to fix this anyway. I know the other actions are created this way, but since this is a new action it seems better to fix it now.